site/cds_rdm/views.py

```diff
@@ -8,19 +8,46 @@
 
 """Additional views."""
 
-from flask import Blueprint, redirect
```

Comment:
I propose to move this a specific migration or redirector module/file as we did in Zenodo [see here](https://github.com/zenodo/zenodo-rdm/blob/master/site/zenodo_rdm/redirector.py).
In this way it will be more clear that this is needed to redirect legacy URLs.

Reply:
If we go with something like migration folder, we could move also the [models](https://github.com/CERNDocumentServer/cds-rdm/blob/master/site/cds_rdm/models.py) from here :)

Reply:
Updated the code with the suggestions :D